### PR TITLE
Correct guide generation command [ci skip]

### PR DESCRIPTION
**Pull request by aditya-kapoor**

### CODE REVIEW

File: guides/rails_guides/generator.rb [hunk 1]
```diff
@@ -28,12 +28,12 @@
...
 #
 #       # generates only association_basics.html
-#       ONLY=assoc ruby rails_guides.rb
+#       ONLY=assoc rake guides:generate
 #
 #     Separate many using commas:
 #
 #       # generates only association_basics.html and migrations.html
-#       ONLY=assoc,migrations ruby rails_guides.rb
+#       ONLY=assoc,migrations rake guides:generate
```
Comment (vipulnsward) [inline guides/rails_guides/generator.rb]: can you change migrations to, say `configuring`, etc. file migrations doesn't exist now.

Comment (aditya-kapoor) [inline guides/rails_guides/generator.rb]: ahhh, I was wondering why it is not generating it.

**Comment by aditya-kapoor** [comment]
@vipulnsward updated PR

**Comment by vipulnsward** [comment]
Thanks!

**Pull request merged**